app/src/main/java/com/berniesanders/connect/MainActivity.java

```diff
@@ -32,8 +48,21 @@ protected void onCreate(Bundle savedInstanceState) {
         mText.setText(mContext.getPackageName());
 
         mActionAlertsModel.getActionAlerts().subscribe(
```

Comment:
@Wopple basically, except using a CompositeSubscription. I can go ahead and implement this right now, just signing on